doc/tntcxx_getting_started.rst

```diff
@@ -0,0 +1,524 @@
+``response.body.error_stack`` or data (tuples)--``response.body.data``.
+In turn, data is a vector of tuples. However, tuples are not decoded and
+come in the form of pointers to the start and the end of msgpacks.
+See the :ref:`"Decoding and reading the data" <gs_cxx_reader>` section to
+understand how to decode tuples.
+
+There are two options for single connection it regards to receiving responses:
+we can either wait for one specific future or for all of them at once.
+We'll try both options in our example. For the ``ping`` request, let's use the
+first option.
+
+.. literalinclude:: ./_includes/Simple.cpp
+   :start-after: doclabel11-1
+   :end-before: doclabel11-2
+   :language: c
+
+For the ``replace`` and ``select`` requests, let's examine the option of
+waiting for both futures at once.
+
+.. literalinclude:: ./_includes/Simple.cpp
+   :start-after: doclabel11-2
+   :end-before: doclabel11-3
+   :language: c
+
+.. // TBD For the detailed information on the connector's API, refer to <link to the API document parts related to the API used here> -- implement this line when API document is ready.
+
+.. _gs_cxx_requests_several:
+
+Several connections at once
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now, let's have a look at the case when we establish two connections
+to Tarantool instance simultaneously.
+
+.. literalinclude:: ./_includes/Simple.cpp
+   :start-after: doclabel11-3
+   :end-before: doclabel11-4
+   :language: c
+
+.. // TBD For the detailed information on the connector's API, refer to <link to the API document parts related to the API used here> -- implement this line when API document is ready.
+
+.. _gs_cxx_requests_close:
+
+Closing connections
+~~~~~~~~~~~~~~~~~~~~
+
+Finally, a user is responsible for closing connections.
+
+.. literalinclude:: ./_includes/Simple.cpp
+   :start-after: doclabel12-1
+   :end-before: doclabel12-2
+   :language: c
+
+.. _gs_cxx_build:
+
+Building and launching C++ application
+-------------------------------------------
+
+Now, we are going to build our example C++ application, launch it
+to connect to Tarantool instance and execute all the requests defined.
+
+Make sure you are in the root directory of the cloned C++ connector repository.
+To build the example application:
+
+.. code-block:: bash
+
+   cd examples
+   cmake .
+   make
+
+Make sure the :ref:`Tarantool session <gs_cxx_prereq_tnt_run>`
+you started earlier is running. Launch the application:
+
+.. code-block:: bash
+
+   ./Simple
+
+As you can see from the execution log, all the connections to Tarantool
+defined in our application have been established and all the requests
+have been executed successfully.
+
+.. //TBD Maybe to quote here the full execution log (62 lines)
+
+.. _gs_cxx_reader:
+
+Decoding and reading the data
+------------------------------
+
+Responses from a Tarantool instance contain raw data, that is, the data encoded
+into the msgpack tuples. To decode client's data, a user has to write one's own
```

Comment:
If it isn't a package, library or CLI program name, we should use the full and capitalized name. 

Also, “one has to write one's own” or “user has to write their own”.
```suggestion
into the MessagePack tuples. To decode client's data, the user has to write their own
```

Reply:
Corrected.